state/clickhouse/clickhouse.go

```diff
@@ -39,12 +39,15 @@ type StateStore struct {
 	config clickhouseMetadata
 }
 
+// maxIdentifierLength is the maximum length for database and table names in ClickHouse
+const maxIdentifierLength = 256
```

Comment:
[Per their docs](https://clickhouse.com/docs/sql-reference/functions/other-functions#getMaxTableNameLengthForDatabase) and [this issue](https://github.com/ClickHouse/ClickHouse/issues/64861) they reference a length of `206` and `250` respectively - are you positive about the `256` limit?

Reply:
paging @middt